.eslintrc.yml

```diff
@@ -43,15 +43,13 @@ settings:
     version: 16.3.2
 
 overrides:
-  - files: '*.md'
-    extends: plugin:mdx/recommended
-
   - files: '*.mdx'
     extends:
       - plugin:mdx/recommended
       - plugin:mdx/overrides
     rules:
       import/no-extraneous-dependencies: 0
+      prettier/prettier: 0
```

Comment:
:raised_eyebrow: I though prettier supported MDX, or does the eslint-mdx-parser interfere with that?

Reply:
Something with newlines that prettier says are missing, but are clearly there. Assuming prettier + eslint-mdx + the beta packages in this project don’t like eachother